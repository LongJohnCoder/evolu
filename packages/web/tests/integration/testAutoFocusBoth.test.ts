import { pageDom, pageGoto } from './helpers';

beforeEach(async () => {
  await pageGoto('testAutoFocusBoth');
});

test('render', async () => {
  await expect(await pageDom()).toMatchSnapshot();
});
