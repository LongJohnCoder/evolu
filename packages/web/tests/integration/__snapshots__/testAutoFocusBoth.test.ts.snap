// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div
  id="__next"
>
  <div
    class="jsx-442929638"
  >
    ·Basic Example·
  </div>
  <div
    autocorrect="off"
    contenteditable="true"
    data-gramm="true"
    role="textbox"
    spellcheck="false"
    style="background-color:#eee;margin-bottom:24px;outline:none;padding:8px;width:300px"
    tabindex="0"
  >
    <div>
      <div
        style="font-size:24px"
      >
        ·heading·
      </div>
      <div
        style="font-size:16px"
      >
        ·paragraph·
      </div>
    </div>
  </div>
  <pre
    style="height:48px;overflow:auto"
  >
    <span
      style="opacity: 1;"
      title="{'children':[{'props':{'style':{'fontSize':'24px'}},'children':['heading']},{'props':{'style':{'fontSize':'16px'}},'children':['paragraph']}]}"
    >
      ·1·
      · ·
      ·{"hasFocus":false}·
    </span>
    <br />
    <br />
    <span
      style="opacity: 0.5;"
      title="{'children':[{'props':{'style':{'fontSize':'24px'}},'children':['heading']},{'props':{'style':{'fontSize':'16px'}},'children':['paragraph']}]}"
    >
      ·0·
      · ·
      ·{"hasFocus":true}·
    </span>
    <br />
    <br />
  </pre>
  <div
    class="jsx-442929638"
  >
    ·Schema Example·
  </div>
  <div
    autocorrect="off"
    contenteditable="true"
    data-gramm="true"
    role="textbox"
    spellcheck="false"
    style="background-color:#eee;margin-bottom:24px;outline:none;padding:8px;width:300px"
    tabindex="0"
  >
    <div>
      <div
        class="jsx-3102962925"
      >
        ·heading·
      </div>
      <div
        class="jsx-1758464876"
      >
        ·paragraph·
      </div>
      <div
        class="jsx-44984361"
      >
        <div
          class="jsx-1758464876"
        >
          ·listitem·
        </div>
      </div>
      <img
        alt="Square placeholder image 80px"
        height="80"
        src="https://via.placeholder.com/80"
        width="80"
      />
    </div>
  </div>
  <pre
    style="height:48px;overflow:auto"
  >
    <span
      style="opacity: 1;"
      title="{'type':'document','children':[{'type':'heading','style':{'font-size':'24px'},'children':['heading']},{'type':'paragraph','style':{'font-size':'16px'},'children':['paragraph']},{'type':'list','style':{'margin':'16px'},'children':[{'type':'listitem','style':{'font-size':'16px'},'children':['listitem']}]},{'type':'image','src':'https://via.placeholder.com/80','alt':'Square placeholder image 80px','width':80,'height':80}]}"
    >
      ·1·
      · ·
      ·{"hasFocus":true,"selection":{"anchor":[0,0,0],"focus":[0,0,0]}}·
    </span>
    <br />
    <br />
    <span
      style="opacity: 0.5;"
      title="{'type':'document','children':[{'type':'heading','style':{'font-size':'24px'},'children':['heading']},{'type':'paragraph','style':{'font-size':'16px'},'children':['paragraph']},{'type':'list','style':{'margin':'16px'},'children':[{'type':'listitem','style':{'font-size':'16px'},'children':['listitem']}]},{'type':'image','src':'https://via.placeholder.com/80','alt':'Square placeholder image 80px','width':80,'height':80}]}"
    >
      ·0·
      · ·
      ·{"hasFocus":true}·
    </span>
    <br />
    <br />
  </pre>
  <button
    class="focus"
    disabled=""
    tabindex="0"
    type="button"
  >
    ·focus·
  </button>
  <button
    class="blur"
    tabindex="-1"
    type="button"
  >
    ·blur·
  </button>
</div>
`;
